zboss/README.rst

```diff
@@ -10,9 +10,12 @@ The nrfxlib repository contains the following versions of the ZBOSS libraries:
 * *Production* version that contains the latest stable ZBOSS libraries.
   This version is enabled with the :option:`CONFIG_ZIGBEE_LIBRARY_PRODUCTION` KConfig option and its files are located in the :file:`zboss/production/` directory.
   This version is enabled by default.
+  The production libraries fully conform to the certification, but they are not necessarily certified.
```

Comment:
I suggest to start a new paragraph here and in line 15/17 (before "This version might not conform ...").